### DISCUSSION TITLE
How to introduce bootstrap's Accordion footer separately

### DISCUSSION THREAD

**Discussion by guiquanH** [twbs/bootstrap#40648] [Q&A]
### Proposal

bootstrap5.2

### Motivation and context

I need to introduce a separate script for the Accordion component, but I can't find a way to do so. I don't want to go and globally introduce all the component code

**Comment by julien-deramond** [comment]
Hello @guiquanH 
I've converted your feature request to a GH discussion first.
Have you tried what is mentioned in [JavaScript > Individual or compiled](https://getbootstrap.com/docs/5.3/getting-started/javascript/#individual-or-compiled)?
